CleverTap/Runtime/Native/UnityNativeWrapper/Models/UnityNativeAppInboxPersistence.cs

```diff
@@ -93,7 +93,15 @@ public static void MarkAsRead(string id)
                 return;
             }
 
-            _preferenceManager.SetInt($"{_readPrefixKey}_{id}", 1);
+            id = $"{_readPrefixKey}_{id}";
```

Comment:
Can you use a new variable? Modifying the `id` method parameter is a bit confusing.